.gitignore

```diff
@@ -11,3 +11,4 @@
 /vector/olm.js
 .DS_Store
 npm-debug.log
+electron/dist
```

Comment:
`/electron/dist`, unless there's a reason for this to work in all subdirs. And please put it in the right place alphabetically :)